CONTRIBUTING.md

```diff
@@ -95,35 +78,35 @@ $ git add my/changed/files
 $ git commit
 ```
 
-### Commit guidelines
-
-Writing good commit logs is important. A commit log should describe what
-changed and why. Follow these guidelines when writing one:
```

Comment:
Maybe we can keep this doc very succinct and put a link right at the top with something like "if you are new or relatively new to contributing to open source, check out this way more informative guide". The link would be to a guide with more general information like this? And it could go into lots of detail, including maybe even some of the stuff from #12744 and material like that?